src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/PKCS1MaskGenerationMethod.cs

```diff
@@ -18,19 +18,13 @@ public PKCS1MaskGenerationMethod()
         public string HashName
         {
             get { return _hashNameValue; }
-            set 
-            {
-                if (value != null && value != "SHA1")
-                {
-                    throw new PlatformNotSupportedException();
-                }
-            }
+            set { _hashNameValue = value ?? "SHA1"; }
```

Comment:
May as well take this and line 15 and fold them into a constant.

I don't really expect us to change this off of SHA-1 as a default, but it'd be nice to change one thing instead of two.